account_bank_statement_import_adyen/models/account_journal.py

```diff
@@ -0,0 +1,15 @@
+from odoo import fields, models
```

Comment:
Can you copy in the copyright stanza from the manifest file to python files where it is missing?